apps/codeforafrica/src/lib/api.netlify-cms/getOurPartners.js

```diff
@@ -25,5 +29,5 @@ export default function geOurPartners(page = "index") {
   const list =
     partnersIds?.map((id) => allPartners.find((p) => p.id === id)) ?? null;
```

Comment:
If we're pulling partners for about page, we need to return **all** partners. I don't think there is even a need for selection in config.